rgw/v2/tests/aws/test_sts_rename_large_object.py

```diff
@@ -111,6 +111,12 @@ def test_exec(config, ssh_con):
         )
     )
     utils.exec_shell_cmd(add_caps_cmd)
+    if admin_user:
+        add_admin_flag = (
+            'sudo radosgw-admin user modify add --uid="{user_id}" --admin true'.format(
```

Comment:
Please check the command,
looks like "add" is invalid option..should be just modify?

Reply:
oh, that was a previous addition. Removed it @ckulal. request a re-review.